### ISSUE TITLE
SSL- Statistics graph won't load

### ISSUE THREAD

**Issue by andyshilton** [ignacionelson/ProjectSend#319]
When logging in, the dashboard displays correctly, but the Statistics block just displays the throbber and "Please wait while the system generates the statistics graph."

Examining things in the console, it appears that it's making a GET to home-statistics.php?days=15 which is returning a 500 Internal Server Error.

**Comment by andyshilton** [comment]
Have now tested this on PHP5.6 and I'm having the same issue. The main thing they both have in common is they are running over SSL.

**Comment by lenamtl** [comment]
Hi, 
could you tried some adjustments from this post about BASE_URI
https://github.com/ignacionelson/ProjectSend/issues/65

**Comment by andyshilton** [comment]
I've edited the <<root>>/sys.var.php file to set the system_uri to use https://<<my-site>>/
I've altered the line in <<root>/index.php Line: 118 from:
  header("location:home.php");
to:
  header("location:".BASE_URI);

I believe that's what #65 appears to be saying to do. Sadly, neither fix the issue. I am still getting 500 error in the console when the dashboard calls https://<<my-site>>/home-statistics.php?days=15

I've tracked it down the SQL statement in home-statistics.php Line 20 - 28.

On an older server running MySQL 5.5 it runs fine. However, on MySQL 5.6, if returns the error: 

"actions_log.timestamp" isn't in GROUP BY

I believe this may have more to do with the stricter controls on timestamps that is also causing issues on installation.
